src/moonlink/src/storage/mooncake_table.rs

```diff
@@ -754,6 +758,11 @@ impl MooncakeTable {
         let non_streaming_batch_id_counter = Arc::new(BatchIdCounter::new(false));
         let streaming_batch_id_counter = Arc::new(BatchIdCounter::new(true));
 
+        // TODO(Paul): Support object storage for WAL
+        let wal_manager_file_config =
+            WalManager::default_wal_storage_config(table_metadata.table_id, &table_metadata.path);
```

Comment:
It breaks recovery.

table path here is meant to store write-through cache, which recreates on recovery / initialization: https://github.com/Mooncake-Labs/moonlink/blob/0ea67cea1e8c1407487e2e5e1fba397d9be5d49d/src/moonlink_connectors/src/pg_replicate/table_init.rs#L64-L65